lib/money-rails/active_record/migration_extensions/options_extractor.rb

```diff
@@ -3,7 +3,12 @@ module ActiveRecord
     module MigrationExtensions
       class OptionsExtractor
         def self.extract(attribute, table_name, accessor, options = {})
-          default = MoneyRails::Configuration.send("#{attribute}_column").merge(options[attribute] || {})
+          # Placing the default_for_both_columns first allows for 
+          # the options of the specific column to take precendence when they are passed. 
+          default_for_both_columns = {}
+          default_for_both_columns =default_for_both_columns.merge(null: options[:null]) if options.has_key?(:null) 
```

Comment:
nit: missing space after `=`

Reply:
Btw, a cleaner approach might be:

```ruby
default_for_both_columns = { null: options[:null] }.compact
```

which will filter out keys with `nil` values

Reply:
Would you like me to improve it?

Reply:
@thebravoman please add the improvements and resubmit

Reply:
@semmons99 Ok. Thanks. I will need some time to get back, because it's been a few months since then.